examples/lighting-app/silabs/src/AppTask.cpp

```diff
@@ -40,7 +40,7 @@
 
 #include <platform/CHIPDeviceLayer.h>
 
-#if (defined(SL_CATALOG_SIMPLE_LED_LED1_PRESENT) || defined(SIWX_917))
+#if (defined(SL_CATALOG_SIMPLE_LED_LED1_PRESENT) || defined(SLI_917_SOC))
```

Comment:
917 defines for for the LEDs shouldn't be necessary anymore since they support the `SL_CTALOG_*` defines now.